server/portal/apps/projects/views.py

```diff
@@ -0,0 +1,263 @@
+"""Project views.
```

Comment:
probably out of scope for this , but we should add unit tests for view.py.  jira ticket?

Reply:
I ended up writing some just to make sure I had the API paths set up correctly.